superset-frontend/package.json

```diff
@@ -236,7 +236,8 @@
     "webpack-bundle-analyzer": "^3.4.1",
     "webpack-cli": "^3.1.1",
     "webpack-dev-server": "^3.1.14",
-    "webpack-sources": "^1.1.0"
+    "webpack-sources": "^1.1.0",
+    "yargs": "12 - 15"
```

Comment:
Replace `minimist` with `yargs` as the latter has better API and is included by other packages anyway. Choose this wide range (`12 -15`) also because it is tested working with the latest version but other packages still depend on v12.

Reply:
`jest-cli` and `webpack-dev-server` uses `12`, `vega-lite` uses `15`. `15` is the latest version, the latest `webpack-dev-server` hasn't upgraded it yet.

Reply:
Wow, [just found out](https://github.com/webpack/webpack-dev-server/pull/2318) there is [a bot ](https://github.com/marketplace/renovate)that automatically upgrade the packages for you, plus migrating all the code. Maybe we can use it to replace dependabot in `superset-ui` and `superset-ui-plugins`.

Reply:
They list it as `dependencies`. The range is specifically to avoid multiple copies. I think it's OK to keep this range for now as we didn't use it too extensively and we already know both v12 and v15 works for our code.